model/generate.xml

```diff
@@ -420,6 +420,20 @@
     <define name="BX_EC_MNEMONIC_TO_SEED_INVALID_IN_LANGUAGES" value="WARNING: The specified words are not a valid mnemonic in any supported dictionary." />
   </command>
 
+  <command symbol="electrum-new" output="string" multipleX="true" category="WALLET" description="Create a mnemonic seed (Electrum) from entropy. WARNING: mnemonic should be created from properly generated entropy.">
+    <option name="prefix" type="electrum" description="The electrum seed type identifier to use. Options are 'standard', 'witness', and 'two_factor_authentication', defaults to 'standard'." />
```

Comment:
Especially for command line (typing) explicitness should be balanced against verbosity, even in these commands where we have single letter shortcuts.  "two_factor_authentication" is a bit much. Maybe something like "dual".